README.md

```diff
@@ -48,4 +48,4 @@ $ kubectl get no -o json | jq ".items[].metadata.labels"
 
 ```
 
-[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery
+[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery/v0.17
```

Comment:
I wonder if we should include the patch version in URL also. i.e v0.17.x. Does it make sense?

Reply:
Good question. The documentation is arranged per release branch. Users always see the documentation of the latest patch release. What you suggested is not supported by the infra.

We don't make available or preserve documentation for each patch release. We could, if there's value for that but that's another story